addons/respawn/XEH_postInit.sqf

```diff
@@ -1,7 +1,11 @@
-// by commy2
 #include "script_component.hpp"
 
 ["ace_rallypointMoved", FUNC(updateRallypoint)] call CBA_fnc_addEventHandler;
 ["unit", FUNC(handlePlayerChanged)] call CBA_fnc_addPlayerEventHandler; // hide enemy rallypoint markers
 
 [QGVAR(showFriendlyFireMessageEvent), FUNC(showFriendlyFireMessage)] call CBA_fnc_addEventHandler;
+
+// fix, because killed EH will not get called at time before respawn EH
+if (GVAR(SavePreDeathGear) && getMissionConfigValue ["respawnDelay", 0] == 0) then {
+    ["CBA_loadingScreenDone", {setPlayerRespawnTime 0.01}] call CBA_fnc_addEventHandler;
+};
```

Comment:
This should be the setting changed code block, so that this fix is applied when changing the setting mid session.

Reply:
It is here https://github.com/shukari/ACE3/blob/0f51226521d3afd6df81d7e2fdd647372df49579/addons/respawn/initSettings.sqf#L9

It needs to execute on startup when SavePreDeathGear is true and also if setting was false and then is set mid mission to true

Miss i something?

Reply:
Is this really needed in postInit then? I also think that even if you use loadingScreenDone, the EH should be added in preInit (with the check for the setting inside the EH code block).

Reply:
Is it important if it is postInit or preInit?

Reply:
changed